### PR TITLE
Adding requestIdleCallback polyfill

### PR DESCRIPTION
**Pull request by nicksorrentino**
to: @ljharb @lencioni

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Included shims:
  - [classlist-polyfill](https://www.npmjs.com/package/classlist-polyfill) - Element.prototype.classList polyfill (only in browsers)
  - [raf](https://www.npmjs.com/package/raf) - requestAnimationFrame polyfill for browsers and node
  - [matchmedia-polyfill](https://github.com/paulirish/matchMedia.js/) - window.matchMedia polyfill (only in browsers)
+ - [requestIdleCallback](https://www.npmjs.com/package/requestidlecallback)
```
Comment (lencioni) [inline README.md]: This should probably be near raf, since they are conceptually pretty similar.

File: package.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
     "ima-babel6-polyfill": "^0.12.0",
     "matchmedia-polyfill": "^0.3.0",
     "raf": "^3.3.0",
+    "requestidlecallback": "^0.3.0",
```
Comment (lencioni) [inline package.json]: The shim you have here looks pretty complex. Did you consider this simpler one? https://github.com/HenrikJoreteg/ric-shim

Comment (nicksorrentino) [reply package.json]: Yeah that one seems reasonable.  I can swap it out

**Comment by nicksorrentino** [comment]
@lencioni updated to ric-shim

**Review by ljharb — CHANGES_REQUESTED** [review]
This doesn't actually shim it - you need to `require` it in `index.js`.

**Comment by nicksorrentino** [comment]
@ljharb you are right!  yesterday I was a bit tired from launch.  Updated now.  Are you okay with assigning it to global like this?

File: index.js [hunk 3]
```diff
@@ -22,3 +22,7 @@ if (typeof window !== 'undefined') {
 }
 
 require('raf/polyfill');
+
+global.requestIdleCallback = require('ric-shim');
```
Comment (ljharb) [inline index.js:26]: it'd be nice to file an upstream issue so that `require('ric-shim/shim')` would do this global installation for us.

**Review by ljharb — APPROVED** [review]
LGTM

**Comment by ljharb** [comment]
Released as v1.4.0

**Pull request merged**